modules/ROOT/pages/import-export-an-organization.adoc

```diff
@@ -1,7 +1,7 @@
 = Install/export an organization
-:description: This page explains what a user with the _Administrator_ profile in Bonita Portal or in the Bonita Administrator Application can see and do about the Organization of Bonita users (an .xml file).
+:description: This page explains what a user with in the Bonita Administrator Application can see and do about the Organization of Bonita users (an .xml file).
 
-This page explains what a user with the _Administrator_ profile in Bonita Portal or in the Bonita Administrator Application can see and do about the Organization of Bonita users (an .xml file).
+This page explains what a user with in the Bonita Administrator Application can see and do about the Organization of Bonita users (an .xml file).
```

Comment:
this page explains what a user with in the Bonita Administrator Application can => this page explains what a user with Administrator profile in the Bonita Administrator Application can

Reply:
Same as above (duplicated so will be replaced by {description})